src/Models/Nonhydrostatic2DModels/nonhydrostatic_2d_model.jl

```diff
@@ -65,6 +64,19 @@ function Models.default_initial_conditions(model::Nonhydrostatic2DModel)
     return Fields.FieldVector(; zero_inits...)
 end
 
+function Models.default_ode_cache(model::Nonhydrostatic2DModel, space_center, space_face)
```

Comment:
Same here. I prefer remove `space_center` and `space_face` from the inputs and get them inside the function.